libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,16 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+# rubocop:disable Documentation
+class Chef
+  # Making the sensitive attribute passive for older chef versions
+  class Resource
+    def sensitive(_ = nil)
```

Comment:
I'm going back and forth here, whether this should be a no-op (since noone uses this on earlier versions of chef), or if we should have this set and retrieve the property like the real version does, so we (and others) could potentially leverage it ourselves.